spring-petclinic-api-gateway/src/main/java/org/springframework/samples/petclinic/api/boundary/web/FallbackController.java

```diff
@@ -0,0 +1,16 @@
+package org.springframework.samples.petclinic.api.boundary.web;
+
+import org.apache.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class FallbackController {
+
+    @RequestMapping("/fallback")
```

Comment:
**suggestion**: add `POST` (and `GET` ?) in the `method` attributes

Reply:
Agreed